### PR TITLE
Update CUDA-Q (formally CUDA Quantum) entries

### PR DESCRIPTION
**Pull request by efratshabtai**
Update CUDA-Q (formally CUDA Quantum) entries

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 
 **C++**
 - [avaloni](https://github.com/avalon-lang/avaloni)  Programming language (interpreter) for classical-quantum hybrid computers.
-- [CUDA Quantum] (https://github.com/NVIDIA/cuda-quantum) - A quantum programming model for heterogeneous quantum-classical workflows.
+- [CUDA-Q] (https://github.com/NVIDIA/cuda-quantum) - Platform for accelerated quantum - classical applications on GPUs, CPUs and QPUs.
```
Comment (dlyongemallo) [inline README.md:41]: I'm not sure why you added a space between "quantum" and "classical" in this line (and in line 50 below), but "quantum-classical" should be a single hyphenated term.

File: README.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [blueqat](https://github.com/Blueqat/Blueqat) - Quantum computing SDK.
 - [Braket](https://github.com/aws/amazon-braket-sdk-python) - [Amazon's](https://aws.amazon.com/braket/) fully managed quantum computing service for building quantum algorithms.
 - [Cirq](https://github.com/quantumlib/Cirq) - Framework for creating, editing, and invoking Noisy Intermediate Scale Quantum (NISQ) circuits.
-- [CUDA Quantum] (https://github.com/NVIDIA/cuda-quantum) C++ support for the CUDA Quantum programming model for heterogeneous quantum-classical workflows.
+- [CUDA-Q] (https://github.com/NVIDIA/cuda-quantum) - Platform for accelerated quantum - classical applications on GPUs, CPUs and QPUs.
```
Comment (dlyongemallo) [inline README.md:50]: Also "quantum-classical" here.

Comment (efratshabtai) [reply README.md]: oh, missed that :)

**Comment by dlyongemallo** [comment]
Thanks @efratshabtai for the contribution!

**Pull request merged**